Test/generate_data_test.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+"""
+>>> from pycm.generate_random_data import (
+...     _generate_class_percentages,
+...     _calculate_class_counts,
+...     generate_confusion_matrix,
+...     generate_confusion_matrix_with_scenario,
+...     Scenario
+... )
+>>> import numpy as np
+>>> from math import isclose
+>>> ABS_TOL = 1e-12
+>>> REL_TOL = 0
+
+# Test for _generate_class_percentages
+>>> assert isclose(_generate_class_percentages(3, Scenario.UNIFORM)[0], 33.333333333333, abs_tol=ABS_TOL)
+>>> assert isclose(_generate_class_percentages(3, Scenario.MAJORITY_CLASS)[0], 166.666666666666, abs_tol=ABS_TOL)
```

Comment:
`166.6`% is a wrong percentage. In the `_generate_class_percentages` function, it does not divide by 100 finally to ensure the sum to `100`. Right now, the majority percentage is calculated as below, which is only working well for value scenarios with higher than `5` classes.
```pycon
majority_percentage = (100/3) * 5 = 166.6
```